force-app/main/default/pages/REL_RelationshipsViewer.page

```diff
@@ -1,187 +1,233 @@
 <apex:page extensions="REL_RelationshipsViewer_CTRL" standardController="Contact" sidebar="false">
+    <apex:includeLightning />
+    <apex:slds />
     <apex:includeScript value="{!URLFOR($Resource.CumulusStaticResources, '/arbor-v0.92/src/graphics/colors.js')}" />
-    <apex:includeScript value="{!URLFOR($Resource.CumulusStaticResources, '/arbor-v0.92/src/graphics/primitives.js')}" />
+    <apex:includeScript
+        value="{!URLFOR($Resource.CumulusStaticResources, '/arbor-v0.92/src/graphics/primitives.js')}" />
     <apex:includeScript value="{!URLFOR($Resource.CumulusStaticResources, '/arbor-v0.92/src/graphics/graphics.js')}" />
     <apex:includeScript value="{!URLFOR($Resource.CumulusStaticResources, '/jquery/jquery-3.5.0.min.js')}" />
     <apex:includeScript value="{!URLFOR($Resource.CumulusStaticResources, '/arbor-v0.92/lib/arbor.js')}" />
     <apex:includeScript value="{!URLFOR($Resource.CumulusStaticResources, '/arbor-v0.92/lib/arbor-tween.js')}" />
 
+    <style>
+        .slds-scope .slds-table tr.slds-hint-parent td, .slds-scope .slds-table tr.slds-hint-parent th {
+            padding: 0.25rem;
```

Comment:
This makes the table more compact, but it won't be included if the component is directly added to a Lightning page or tab (which I guess is fine, just pointing it out 🤷‍♂️ )

Reply:
Yep, this was a catch by Grace who noticed how rows in this table were about 20-30% taller when embedded in a Visualforce page.